src/Components/Facility/ConsultationForm.tsx

```diff
@@ -187,11 +236,17 @@ export const ConsultationForm = (props: any) => {
       setIsLoading(true);
       const data = {
         symptoms: state.form.symptoms,
-        other_symptoms: state.form.otherSymptom ? state.form.other_symptoms : undefined,
-        symptoms_onset_date: state.form.hasSymptom ? state.form.symptoms_onset_date : undefined,
+        other_symptoms: state.form.otherSymptom
+          ? state.form.other_symptoms
+          : undefined,
+        symptoms_onset_date: state.form.hasSymptom
```

Comment:
Codacy found an issue: [Identifier 'symptoms_onset_date' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5808151)